service/controller/drainer.go

```diff
@@ -9,9 +9,10 @@ import (
 	"k8s.io/apimachinery/pkg/labels"
 	"k8s.io/apimachinery/pkg/runtime"
 
+	"github.com/giantswarm/kvm-operator/service/controller/key"
```

Comment:
Is this intentional?

Reply:
That looks strange. Goimports should have caught that. Let me see.

Reply:
Fixed.